### PR TITLE
RHDEVDOCS-4496: Added the feature exporting applications to another p…

### PR DESCRIPTION
**Pull request by shipsing**
Purpose: To resolve https://issues.redhat.com/browse/RHDEVDOCS-4496
Version(s): enterprise-4.12 only

Link to docs preview: https://50950--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-web-console

SME review :@debsmita1
QE review: @sanketpathak 
Peer review: @bburt-rh

### CODE REVIEW

**Comment by shipsing** [comment]
Hello @debsmita1, Could you please help with SME review of this PR?

Thanks

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-web-console"]
 === Web console
 
+[id="ocp-4-12-developer-perspective"]
+==== Developer Perspective
+
+* With this update, you can export your application in the ZIP file format to another project or cluster using the *+Add* view or the *Topology* view.
```
Comment (debsmita1) [inline release_notes/ocp-4-12-release-notes.adoc:65]: may be ?
```suggestion
* With this update, you can export your application in the ZIP file format to another project or cluster using the newly added `Export application` card on the *+Add* page or the `Export Application` button on the *Topology* view. 
```
And the `Export Application` option was already there on the topology page, we simply added a card to the Add page

Comment (shipsing) [reply release_notes/ocp-4-12-release-notes.adoc]: @debsmita1, I have tweaked the line a bit. Also, if the Export application buuton is not newly added in the topology view with this release, we can skip that part as we have to highlight only the new features.

**Comment by shipsing** [comment]
@sanketpathak, Could you please help with QE review of this PR? Thanks

**Comment by shipsing** [comment]
@sanketpathak, Could you please help with QE review of this PR? Thanks

**Comment by sanketpathak** [comment]
LGTM

**Comment by bburt-rh** [comment]
lgtm

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**